src/loaders/OmeZarrLoader.ts

```diff
@@ -156,7 +160,13 @@ class OMEZarrLoader implements IVolumeLoader {
       }
     }
 
-    const downsampleZ = 2; // z/downsampleZ is number of z slices in reduced volume
+    const optimalLevel = estimateLevelForAtlas(spatialDims, 2048);
+    // assume all levels are decreasing in size.  If a larger level is optimal then use it:
+    if (optimalLevel < levelToLoad) {
+      levelToLoad = optimalLevel;
+    }
+
+    const downsampleZ = 1; // z/downsampleZ is number of z slices in reduced volume
```

Comment:
previous zarr loading code was only loading half the z slices... not what we want to do just yet.